examples/north_sea/plot_setup.py

```diff
@@ -0,0 +1,59 @@
+from model_config import *
+import matplotlib.pyplot as plt
+import numpy
+
+
+# Plot mesh
+fig, axes = plt.subplots(figsize=(5.4, 4.8))
+mesh2d = Mesh("north_sea.msh")
+triplot(
+    mesh2d,
+    axes=axes,
+    interior_kw={"linewidth": 0.5, "edgecolor": "gray"},
+    boundary_kw={"linewidth": 0.8},
+)
+axes.axis(False)
+axes.legend()
+plt.tight_layout()
+plt.savefig("north_sea_mesh.png", dpi=300)
+
+# Plot bathymetry in a logarithmic colourmap
+P1_2d = get_functionspace(mesh2d, "CG", 1)
+bathymetry2d = Function(P1_2d)
+interpolate_bathymetry(bathymetry2d)
```

Comment:
This needs the `etopo1.nc` file which is not included in the repository. So the user cannot excecute this script. Maybe just plot the included `north_sea_bathymetry.h5` data instead?

Reply:
Good point